modules/priceFloors.js

```diff
@@ -286,9 +286,9 @@ export function updateAdUnitsForAuction(adUnits, floorData, auctionId) {
       bid.auctionId = auctionId;
       bid.floorData = {
         skipped: floorData.skipped,
-        modelVersion: utils.deepAccess(floorData, 'data.modelVersion'),
-        location: utils.deepAccess(floorData, 'data.location'),
         skipRate: floorData.skipRate,
+        modelVersion: utils.deepAccess(floorData, 'data.modelVersion'),
+        location: utils.deepAccess(floorData, 'data.location', 'noData'),
         fetchStatus: _floorsConfig.fetchStatus
```

Comment:
We should pass along `floorProvider` here.

The point of `floorProvider` is so analytics adapters can pick and choose which data they send to their endpoints.